### PR TITLE
Feature: Security Alerts

### PR DESCRIPTION
**Pull request by JanKrb**
Solving issue https://github.com/KrbDevelopment/Articly-Backend/issues/3

### CODE REVIEW

File: app/Http/Controllers/API/User/AuthController.php [hunk 1]
```diff
@@ -321,9 +321,15 @@ public function update_details(Request $request, $account_id) {
             return $this->sendError('Invalid user', ['user_id' => $account_id]);
         }
 
+        $old_email = $account->email;
+
         $account->update($request->all());
         $account->sendActivity('Account details has been changed', 'The profile details has been changed through the profile or an admin');
 
+        if ($old_email !== $request->get('email')) {
+            $account->sendEmailChangedNotification();
```
Comment (NinoKrb) [inline app/Http/Controllers/API/User/AuthController.php:340]: Please have the email changed message sent to the old email address instead of the new one.

Comment (JanKrb) [reply app/Http/Controllers/API/User/AuthController.php]: Should be fixed in https://github.com/KrbDevelopment/Articly-Backend/commit/4dd87e012e5c7c0e10c5883349493078374b594f

File: app/Http/Controllers/API/User/AuthController.php [hunk 2]
```diff
@@ -331,13 +331,18 @@ public function update_details(Request $request, $account_id) {
             return $this->sendError('Invalid user', ['user_id' => $account_id]);
         }
 
-        $old_email = $account->email;
+        $changedEmail = $account->email !== $request->get('email');
+        if ($changedEmail) {
+            $account->sendEmailChangedNotification();
```
Comment (NinoKrb) [inline app/Http/Controllers/API/User/AuthController.php:336]: Please send the user an email that he should verify his email

Comment (JanKrb) [reply app/Http/Controllers/API/User/AuthController.php:336]: Fixed in https://github.com/KrbDevelopment/Articly-Backend/commit/767f18e91cf4f2eb5917e27b82829c75707bca4a

**Pull request merged**